Source/CesiumRuntime/Public/CesiumGeoreferenceListener.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2021 CesiumGS, Inc. and Contributors
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/Interface.h"
+#include "CesiumGeoreferenceListener.generated.h"
+
+// This class does not need to be modified.
+UINTERFACE(MinimalAPI)
+class UCesiumGeoreferenceListener : public UInterface {
+  GENERATED_BODY()
+};
+
+/**
+ *
+ */
+class CESIUMRUNTIME_API ICesiumGeoreferenceListener {
+  GENERATED_BODY()
+
+public:
+  /**
+   * Notifies georeferenced objects that the underlying georeference transforms
+   * have been updated. The new transforms can be fetched from the
+   * `CesiumWorldGeoreference` actor.
```

Comment:
This *might* be `CesiumGeoreference`, but ... this wouldn't necessarily tell the user where to obtain the transforms. Some options for "tansporting" this information range from

1, You have to store a reference to the GeoRef, and fetch the info from that reference on your own
2. The method could receive the `GeoRef*` that was updated (i.e. the one that triggered that call)
3. The method could receive ~"something like the updated transform"

For 1., the receiver of this call would not know which GeoRef was updated. For 3. it's not clear *which* information that is (it's more than a 4x4 matrix, for sure). In other contexts, I'd pragmatically go for 2. (and consider something like a `GeoRefChangedEvent` instead, if more information would have to be transported through that call).

But for 2., the caller could once again *store* this reference that he receives there, raising further questions about its validity and such. Depending on the other decisions, the method could also receive "an identifier" for the GeoRef, like `NotifyGeoreferenceUpdated("THE_DEFAULT_ONE")` vs. `NotifyGeoreferenceUpdated("MY_SPECIAL_ONE")`, where this "identifier" is used to access some sort of global/singleton-like instance.

(tl;dr: This has to be sorted out, and aligned with the (coarse-grained) GeoRef refactoring....)